hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/store/driver/impl/StateStoreFileBaseImpl.java

```diff
@@ -227,7 +270,23 @@ public <T extends BaseRecord> QueryResult<T> get(Class<T> clazz)
     if (metrics != null) {
       metrics.addRead(monotonicNow() - start);
     }
-    return new QueryResult<T>(ret, getTime());
+    return new QueryResult<>(result, getTime());
+  }
+
+  private <T extends BaseRecord> Void getRecordsFromFileAndRemoveOldTmpRecords(Class<T> clazz,
```

Comment:
Can you add documentation to this function indication that the results list is being modified to collect the results.
Changing the function name would make that clearer too.

Reply:
Done, thanks

Reply:
Looks good.